vhdbuilder/parts/k8s/cloud-init/artifacts/kubelet.service

```diff
@@ -1,40 +0,0 @@
-[Unit]
-Description=Kubelet
-ConditionPathExists=/usr/local/bin/kubelet
-{{if EnableEncryptionWithExternalKms}}
-Requires=kms.service
-{{end}}
-
-[Service]
-Restart=always
-EnvironmentFile=/etc/default/kubelet
-SuccessExitStatus=143
-ExecStartPre=/bin/bash /opt/azure/containers/kubelet.sh
-ExecStartPre=/bin/mkdir -p /var/lib/kubelet
-ExecStartPre=/bin/mkdir -p /var/lib/cni
-ExecStartPre=/bin/bash -c "if [ $(mount | grep \"/var/lib/kubelet\" | wc -l) -le 0 ] ; then /bin/mount --bind /var/lib/kubelet /var/lib/kubelet ; fi"
-ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
-{{/* This is a partial workaround to this upstream Kubernetes issue: */}}
-{{/* https://github.com/kubernetes/kubernetes/issues/41916#issuecomment-312428731 */}}
-ExecStartPre=/sbin/sysctl -w net.ipv4.tcp_retries2=8
-ExecStartPre=/sbin/sysctl -w net.core.somaxconn=16384
-ExecStartPre=/sbin/sysctl -w net.ipv4.tcp_max_syn_backlog=16384
-ExecStartPre=/sbin/sysctl -w net.core.message_cost=40
-ExecStartPre=/sbin/sysctl -w net.core.message_burst=80
-
-ExecStartPre=/sbin/sysctl -w net.ipv4.neigh.default.gc_thresh1=4096
```

Comment:
I notice this is different than the one in parts folder

Reply:
--- a/parts/linux/cloud-init/artifacts/kubelet.service
+++ b/parts/linux/cloud-init/artifacts/kubelet.service
@@ -22,9 +22,9 @@ ExecStartPre=/sbin/sysctl -w net.ipv4.tcp_max_syn_backlog=16384
 ExecStartPre=/sbin/sysctl -w net.core.message_cost=40
 ExecStartPre=/sbin/sysctl -w net.core.message_burst=80
 
-ExecStartPre=/bin/bash -c "if [ $(nproc) -gt 8 ]; then /sbin/sysctl -w net.ipv4.neigh.default.gc_thresh1=4096; fi"
-ExecStartPre=/bin/bash -c "if [ $(nproc) -gt 8 ]; then /sbin/sysctl -w net.ipv4.neigh.default.gc_thresh2=8192; fi"
-ExecStartPre=/bin/bash -c "if [ $(nproc) -gt 8 ]; then /sbin/sysctl -w net.ipv4.neigh.default.gc_thresh3=16384; fi"
+ExecStartPre=/sbin/sysctl -w net.ipv4.neigh.default.gc_thresh1=4096
+ExecStartPre=/sbin/sysctl -w net.ipv4.neigh.default.gc_thresh2=8192
+ExecStartPre=/sbin/sysctl -w net.ipv4.neigh.default.gc_thresh3=16384

we need to override the one in cse.

Reply:
hmm lemme double check

Reply:
Good catch, thank you for that,